operator/src/runners/fixtures/membership.rs

```diff
@@ -66,8 +66,14 @@ pub async fn run(args: MembershipCmd) -> anyhow::Result<()> {
         .seconds
         .try_into()?;
 
-    let trusted_client_state =
-        trusted_light_block.to_sol_client_state(args.trust_level.try_into()?, unbonding_period)?;
+    // Defaults to the recommended TrustingPeriod: 2/3 of the UnbondingPeriod
```

Comment:
Btw, where does this default value come from?

Reply:
I've seen it used other places, but I found it when reading up on the value here: https://tutorials.cosmos.network/academy/3-ibc/4-clients.html

The CometBFT docs only says "This value should be significantly smaller than the unbonding period." 🤷